### PR TITLE
compensate for viewer padding if any

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce @dvoytenko PTAL

File: extensions/amp-user-notification/0.1/amp-user-notification.js [hunk 1]
```diff
@@ -257,7 +261,23 @@ export class AmpUserNotification extends AMP.BaseElement {
...
+    const viewerPaddingTop = this.viewer_.getPaddingTop();
+
+    // if there is no viewer padding, then no need to do any offset
+    if (viewerPaddingTop > 0) {
+      // we need to use getComputedStyle in cases where `top` is not in pixels
+      const currentTop = parseFloat(this.win_
+          ./*REVIEW*/getComputedStyle(this.element, null)
+          .getPropertyValue('top'));
+
+      const top = Math.max(viewerPaddingTop, currentTop);
```
Comment (dvoytenko) [inline extensions/amp-user-notification/0.1/amp-user-notification.js]: This assumes that `top` is defined in `px`. Also, I believe it should be `paddingTop + top`, no? Please confirm. IN the meantime, I'm gonna go ahead and try to do this via `FixedLayer` - so please don't rush fixing. Just let me know how you think it should work.

Comment (erwinmombay) [inline extensions/amp-user-notification/0.1/amp-user-notification.js]: @dvoytenko this is why we use getComputedStyle which will return the value in px even if it was declared in em for example (tried it a couple of times on chrome and that seems to be the behavior, although i guess i have to guard for browser support here).

on the 2nd part, you are right it should be paddingTop + top. ill update

**Comment by erwinmombay** [comment]
closing in favor of @dvoytenko's work.

**Pull request closed without merging**